credential-exchange-types/src/format.rs

```diff
@@ -42,22 +44,19 @@ pub struct Account<E = ()> {
     pub id: B64Url,
     /// A pseudonym defined by the user to name their account. If none is set, this should be an
     /// empty string.
-    pub user_name: String,
+    pub username: String,
     /// The email used to register the account in the previous provider.
     pub email: String,
-    /// This OPTIONAL field holds the user’s full name.
+    /// This field holds the user’s full name.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub full_name: Option<String>,
```

Comment:
Question, now that the OPTIONAL and REQUIRED are removed from the comments, do we assume that any field wrapped in `Option<>` is optional and everything else is required?

Reply:
Yes, I thought that it didn't make much sense to keep type information in comments.